src/main/java/seedu/duke/ui/Message.java

```diff
@@ -3,4 +3,16 @@
 public final class Message {
     public static final String ERROR_UNKNOWN =
             "Sorry, something went wrong.";
+
+    public static final String ERROR_INDEX_OUT_OF_BOUNDS =
+            "Index is out of bounds";
+
+    public static final String ERROR_NUMBER_FORMAT =
+            "Index is not a number";
+
+    public static final String ERROR_TYPE_TO_ADD_NOT_SPECIFIED =

```

Comment:
Consider shortening the variable name, the hard limit is 30 characters. [Rationale](https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#not-too-long-not-too-short).